indexer-app/src/cmr/indexer/api/routes.clj

```diff
@@ -47,7 +47,7 @@
       ;; Index a concept
       (POST "/" {body :body context :request-context params :params headers :headers}
         (let [{:keys [concept-id revision-id]} (walk/keywordize-keys body)
-              options {:ignore_conflict? (ignore-conflict? params)}]
+              options {:ignore-conflict? (ignore-conflict? params)}]
```

Comment:
We should be able to test this with an integration test.

1.) Ingest a collection and then update that collection so that you have two revisions.
2.) Call this endpoint to index revision 1 with ignore_conflict false and see that it returns a 409.
3.) Call this endpoint to index revision 1 with ignore_conflict true and see that it returns a 2xx status code.

Reply:
That's what I did in SIT.